examples/todolist/command_handler/command_handler.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
```

Comment:
hpp extensions only for templates or self-sustaining header.
use .h for regular headers